api/PclusterApiHandler.py

```diff
@@ -41,12 +48,20 @@
         USER_POOL_ID = secret.get("userPoolId")
         CLIENT_ID = secret.get("clientId")
         CLIENT_SECRET = secret.get("clientSecret")
+        if not SCOPES_LIST:
+            SCOPES_LIST = "openid"
+        elif "openid" not in SCOPES_LIST:
+            SCOPES_LIST += " openid"
+        if not AUTH_URL:
+            AUTH_URL = f"{AUTH_PATH}/login?response_type=code&client_id={CLIENT_ID}&scope={SCOPES_LIST}&redirect_uri={REDIRECT_URL}"
+        if not JWKS_URL:
+            JWKS_URL = os.getenv("JWKS_URL",
+                                 f"https://cognito-idp.{REGION}.amazonaws.com/{USER_POOL_ID}/" ".well-known/jwks.json")
```

Comment:
These setups should be outside of the if since this if branch only covers the case that cognito info (userpool id, client-id and client-secret) are stored in secrets manager.

Plus, the AUTH_PATH is not needed and AUTH_URL should cover the whole URL to use for user redirection, only missing the query params part (so from the '?' on)